### DISCUSSION TITLE
Ritraduzione jack-tutorial

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#40] [Revisione]
sto ritraducendo tutto jack-tutorial e sono arrivato qui a questo pezzo, ho quasi finito, ma c'è un intoppo, 
facendo un confronto, mi sono accorto che mancano pezzi nella versione italiana....
come devo comportarmi?

....da jack-tutorial

_originale dal file in inglese_

{	881	}{	If you make it back, stop in at the Last Round - it's this bar downtown here - I'll fill you in on the politics. [chuckle]Now that's the stuff that'll kill ya. [laugh]Good luck.	}{	If 

you make it back, stop in at the Last Round - it's this bar downtown here - I'll fill you in on the politics. [chuckle]Now that's the stuff that'll kill ya. [laugh]Good luck.	}{	#	}{	}{	}{	

}{	}{	}{	}{	}{	}{	}
{	882	}{	I'll see you there.	}{	I'll see you there.	}{	0	}{	G.Patch_Plus == 0	}{	G.Tut_Jack = 15	}{	}{	}{	

}{	}{	}{	}{	We will be there soon.	}
{	883	}{	I'll see you there after I got a good day's sleep.	}{	I'll see you there after I got a good day's sleep.	}{	0	}{	G.Patch_Plus == 1	}{	

G.Tut_Jack = 15	}{	}{	}{	}{	}{	}{	}{	We will be there after we close our eyes during the daylight.	}
{	884	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}

-----------------------------------------------------------

_la mia traduzione_

{	881	}{	Se torni indietro, fermati al Last Round - e' quel bar in centro qui - Ti aggiornero' sulla politica. [ridacchia]Ora quella e' la roba che ti uccidera'.[risata] Buona fortuna.	}{	Se 

torni indietro, fermati al Last Round - e' quel bar in centro qui - Ti aggiornero' sulla politica.[ridacchia] Ora quella e' la roba che ti uccidera'.[risata] Buona fortuna.	}{	#	}{	}{	}{	

}{	}{	}{	}{	}{	}{	}
{	882	}{	Ci vediamo li'.	}{	Ci vediamo li'.	}{	0	}{	G.Patch_Plus == 0	}{	G.Tut_Jack = 15	}{	}{	}{	

}{	}{	}{	}{	Saremo li' presto.	}
{	883	}{	Ci vediamo li' dopo che Mi faccio una giornata di sonno.	}{	Ci vediamo li' dopo che Mi faccio una giornata di sonno.	}{	0	}{	G.Patch_Plus == 

1	}{	G.Tut_Jack = 15	}{	}{	}{	}{	}{	}{	}{	Saremo li' dopo aver chiuso gli occhi durante la luce del giorno.	}
{	884	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}{	}

----------------------------------------------------------

**_questo invece e' il file della traduzione ufficiale presa da voi_**
	
{	881	}{	Se ce la fai a tornare indietro, fermata al Last Round - e' il bar a downtown - ti diro' di piu' sulla politica. Ora anche questa roba puo' ucciderti. Buona fortuna.	}{	Se 

ce la fai a tornare indietro, fermata al Last Round - e' il bar a downtown - ti diro' di piu' sulla politica. Ora anche questa roba puo' ucciderti. Buona fortuna.	}{	#	}{		}{		

}{	}{	}{	}{	}{	}{	}{		}
{	882	}{	(Auto-End)	}{	(Auto-End)	}{	0	}{	G.Patch_Plus == 

0	}{	G.Tut_Jack = 15	}{	}{	}{	}{	}{	}{	}{	(Auto-End)	}
{	883	}{	Ci vediamo li' dopo aver dormito bene.	

}{	Ci vediamo li' dopo aver dormito bene.	}{	0	}{	G.Patch_Plus == 1	}{	G.Tut_Jack = 15	}{	}{	}{	}{	}{	}{	

}{	Saremo li' dopo aver chiuso gli occhi durante il giorno.	}
{	884	}{		}{		}{		}{		}{

--------------------------------------------------------

**che succede? manca qualcosa, o qualcosa è stato tagliato per motivi a me oscuri?

linea 882, mancano 3 piccole frasi!

Fate brillare una luce e dare spiegazioni, prego!**

_Segnalo inoltre che nell'intro iniziale dove parla il principe, ci sono 3 frasi di Nines che nei sottotitoli, non sono state tradotte.

poi dando un guardata qui e li, mi sono saltate all'occhio alcune frasi. (non ho letto tutto, ho solo dato una sbirciata random)

se apprezzato, propongo queste traduzioni alternative :_

I'll go against all my better judgment
Andrò contro tutto il mio miglior giudizio

simple pickup's all that's involved.	
semplicemente raccogli tutto ciò che è coinvolto.

Hmm mmm mmm, cram met time crab nipple meh.
Hmm mmm mmm, spassiamocela strizzando capezzoli mah.

Hey, crab nipple this 'n crab nipple that, 'n...
Ehi, strizziamo questo capezzolo e strizziamo quest'altro, e...
oppure
Ehi, ti strizzo questo capezzolo e ti strizzo quest'altro, e...

si potrebbe essere anche piu specifici e precisi se si saprebbe il contesto esatto
in cui tutte queste tipologie di  frasi "capezzolose" occorrono.

continuo guardando qui e li.....

I'd be pissed if this wasn't so cute. Look at you trying so hard; it's absolutely precious!" 
Sarei incazzato se non fosse così affascinante. Guardarti che ci provi così tanto; è assolutamente apprezzabile!"

Yeah. Whenever you're done sweetcheeks, we can move this show along.
Sì. Quando che hai finito, dolcezza, possiamo proseguire questo spettacolo.

_se qualcuno mi legge, battesse un colpo...._

**Comment by tesivo** [comment]
Ho controllato ed è come dici tu, mancano delle frasi, ma di un tipo di mancanza meno grave(nel senso che, a parte le linee di dialogo mancanti, non viene danneggiata alcuna quest del gioco), nel senso che invece del comando (Auto-End), che quando è presente nella versione inglese non va tradotto, dovevano esserci delle frasi specifiche.

Ottimo per le tre frasi si Nines, appena riesco le cerco e le traduco ;)

Apprezziamo tutte le migliorie ;) fai riferimento a quale versione inglese, la 10.7 presa da dove? (devo ritrovare le righe a cui fai riferimento)

Perdonami se talvolta passa più tempo del solito prima che arrivi una mia risposta

Quando hai delle traduzioni che vuoi caricare puoi procedere in più modi:
- carichi la tua traduzione qui su github
- carichi i file dove preferisci e poi qualcuno di noi la carica su github
- (sconsigliata) metti frase per frase la traduzione. Tieni presente che devi dirci di che file e riga stai parlando e sarebbe utile avere, in questo caso, sia file-riga inglese che file-riga italiano

**Comment by NarakuITA** [comment]
> Ho controllato ed è come dici tu, mancano delle frasi, ma di un tipo di mancanza meno grave(nel senso che, a parte le linee di dialogo mancanti, non viene danneggiata alcuna quest del gioco), nel senso che invece del comando (Auto-End), che quando è presente nella versione inglese non va tradotto, dovevano esserci delle frasi specifiche.
> 
> Ottimo per le tre frasi si Nines, appena riesco le cerco e le traduco ;)
> 
> Apprezziamo tutte le migliorie ;) fai riferimento a quale versione inglese, la 10.7 presa da dove? (devo ritrovare le righe a cui fai riferimento)
> 
> Perdonami se talvolta passa più tempo del solito prima che arrivi una mia risposta
> 
> Quando hai delle traduzioni che vuoi caricare puoi procedere in più modi:
> 
> * carichi la tua traduzione qui su github
> * carichi i file dove preferisci e poi qualcuno di noi la carica su github
> * (sconsigliata) metti frase per frase la traduzione. Tieni presente che devi dirci di che file e riga stai parlando e sarebbe utile avere, in questo caso, sia file-riga inglese che file-riga italiano

Secondo me fa prima, come consigliato nell'altra discussione, caricare tutto su un file compresso e lo passa qui. Tu poi prederai i file revisionati e li inserisci nel pacchetto della traduzione.

**Comment by tesivo** [comment]
Si, la prima opzione se vuole cimentarsi con git. La seconda dovrebbe essere quella più veloce e già pronta. La terza opzione temo possa creare solo confusione.

**Comment by crazyLillo** [comment]
per quello che lavoro sul file originale inglese della 10.7

ok per le frasi di nines, che tanto piu in la ho intenzione di ritradurre la intro.

scusa se le segnalazioni random non avevano indicazioni, mi ero concentrato sul contenuto.

tu scrivi: 

carichi la tua traduzione qui su github (COME?)
carichi i file dove preferisci e poi qualcuno di noi la carica su github (SPIEGA!)

Come dice NarakuITA, posso caricare un file compresso, ci proverò.

per esempio, ho finito di ritradurre jack-tutorial.
mi manca solo di ricomporre il file, (mi son diviso il lavoro in pezzi, cosa mia)
una volta ricomposto il file (dlg), lo provo, poi lo copio in una cartella, lo comprimo e lo mando qui aprendo un nuovo issue.ok

**Comment by tesivo** [comment]
Ottimo.

A te la scelta sul come caricare la traduzione, se vuoi puoi fare qui su GitHub, se vuoi su GameTranslator, se vuoi altrove, dimmi tu cosa preferisci. Postare il testo tradotto qui non funziona molto bene perchè si perde la formattazione e si fatica ad effettuare l'aggiornamento della traduzione.

Sì, se via file compresso ti va bene così sia! Ok rimango in attesa

**Comment by crazyLillo** [comment]
gia postato il file compresso 20 ore prima del tuo messaggio, non lo hai visto...he he he questo è overdose da studio...

**Comment by NarakuITA** [comment]
> Ottimo.
> 
> A te la scelta sul come caricare la traduzione, se vuoi puoi fare qui su GitHub, se vuoi su GameTranslator, se vuoi altrove, dimmi tu cosa preferisci. Postare il testo tradotto qui non funziona molto bene perchè si perde la formattazione e si fatica ad effettuare l'aggiornamento della traduzione.
> 
> Sì, se via file compresso ti va bene così sia! Ok rimango in attesa

Vedi se riesci a spostare tutte ti questi Topic, che non sono problemi, nell'area discussioni. Così è più Ordinato.

**Comment by crazyLillo** [comment]
io non ho la piu pallida idea di come si faccia a spostare topic

**Reply by NarakuITA** [reply]
> io non ho la piu pallida idea di come si faccia a spostare topic

Tranquillo, il Topic è essenzialmente una "discussione". Ho usato Topic come parola, per separarla dalla sezione "discussioni. Sennò avrei dovuto dire: Sposta la discussione in discussioni. Non mi piaceva tanto. ehehhe

**Comment by tesivo** [comment]
Spostato
